CONFIGURATION.md

```diff
@@ -136,6 +136,7 @@ batch.size                               |  P  | 1 .. 2147483647 |       1000000
 delivery.report.only.error               |  P  | true, false     |         false | low        | Only provide delivery reports for failed messages. <br>*Type: boolean*
 dr_cb                                    |  P  |                 |               | low        | Delivery report callback (set with rd_kafka_conf_set_dr_cb()) <br>*Type: see dedicated API*
 dr_msg_cb                                |  P  |                 |               | low        | Delivery report callback (set with rd_kafka_conf_set_dr_msg_cb()) <br>*Type: see dedicated API*
+sticky.linger.ms                         |  P  | -1 .. 900000    |            -1 | high       | Delay in milliseconds to wait to assign new sticky partitions for each topic. By default, sticky behavior is disabled (set to *-1*), maintaining the original behavior of all partitioners. A higher value allows for more effective batching (less overhead) of null keyed messages at the expense of fine grain load balancing. This behavior affects messages with the key NULL in all cases, and messages with key lengths of zero when the consistent_random partitioner is in use. <br>*Type: integer*
```

Comment:
Change off-value to 0, and set the default to linger.ms * magical_value. Also make it so that this property is automatically set to linger.ms * magical_value if this property is not explicitly set (see conf_finalize()).